README.rst

```diff
@@ -0,0 +1,217 @@
+This is a userscript intended to be used with greasemonkey or tampermonkey.
+
+It enriches the github pull request page with so called 'incremental diffs'.
+This means that you will get a diff for changes pushed to that pull request,
+showing the difference before and after the push.
+
+This is particular useful for situations where the reviewee was asked to do some
+changes in their existing commits (not additional commits). The reviewer can
+then use this to see those changes directly, not having to go through all the
+commits again, making sure the requested changes were addressed.
+
+All this works by using github webhook, an external server and this script.
+
+User Setup
+==========
+
+A working installation of greasemoneky or tampermonkey is required.
+
+To install the script, just follow this `link <https://raw.githubusercontent.com/sociomantic-tsunami/kelpie/master/incremental-pr-diffs.user.js>`_.
+
+After navigating to any pull request page, the script will ask you to setup a
+personalized access token for github so it can access the github API.
+
+The hash data url is optional and not required for the normal user. If you do
+need it, your repository admin should have told you about it.
+
+You can change your configuration settings later anytime by double-clicking on
+the ``Incremental Diffs Active`` headline at the bottom of the right sidebar.
```

Comment:
Ah, maybe "is Active"? I thought it was a typo for "Archive" or something. Since now that's only used to (re)configure it, maybe change it to "Configure Incremental Diffs" (and maybe a button).